### ISSUE TITLE
[core][feature]:Add scroll to top Button 

### ISSUE THREAD

**Issue by parthksingh1** [enrich-platforms/toppings#2]
## Issue Type
- [X] Feature Request

## Description
Enhanced User Experience: Users don't need to manually scroll upwards, which can be tedious on very long pages.
Improved Navigation: Provides an easy and quick way to access the top menu or any other content located at the top.
Adaptive Design: Can be designed to appear only after the user has scrolled down a specific amount, ensuring that it doesn't interfere with the initial page view.

**Comment by parthksingh1** [comment]
Please Assign this issue to me will implement this and will make website look more attractive

**Comment by DarhkVoyd** [comment]
Hi @parthksingh1!

Adding a "scroll to top" button is an excellent enhancement. To ensure it blends seamlessly with web pages, let's discuss the design to make it visually pleasing and adaptive. Familiarizing yourself with our code conventions will help maintain uniformity throughout the extension.

I've assigned you to this issue and added relevant labels, aligning with our Hacktoberfest event. Please make sure your code is linted, compiles, and runs locally before creating a PR.

A good place to work on the implementation would be at the `core` script [here](https://github.com/enrich-platforms/toppings/blob/8e04ada621e23515f45b4a65ec7c64d5f1867e02/src/scripts/core.ts).

Let me know if I can help in any way, feel free to open a draft PR for me to look at if you get stuck.

**Comment by parthksingh1** [comment]
Hello @DarhkVoyd ,
I hope this message finds you well. I recently cloned the repository and was trying to set it up locally. I noticed that while the package.json has scripts for building the project with Webpack, there isn't a clear script or instruction for running the project on a local development server.

Could you provide some guidance on how to properly run the project locally? Are there any specific steps or additional configurations that I should be aware of? It would be great if this could be included in the repository's README or documentation to assist others in the future.

Thank you in advance for your assistance, and I appreciate the work you've done on this project!

**Comment by DarhkVoyd** [comment]
@parthksingh1 Thanks for pointing that out! I have updated the [Contributing Guide](https://github.com/enrich-platforms/toppings/blob/main/CONTRIBUTING.md) with clear instructions for running the project locally. Feel free to check it out.

**Comment by AmitSaha15** [comment]
@DarhkVoyd can I try this? The assignee hasn't made any PR so...

**Comment by DarhkVoyd** [comment]
@AmitSaha15 I recommend waiting for a brief period to allow the assignee to respond. If they don't reply within a reasonable timeframe (for example, a few more days), feel free to pick up the issue and start working on it.

**Comment by AmitSaha15** [comment]
@DarhkVoyd yes sure!

**Comment by DarhkVoyd** [comment]
In alignment with our broader initiative to introduce modular components within the extension, we have decided to close this issue and transition the task to a new, dedicated issue focused on the implementation of the "ScrollToTop" component.

This strategic approach allows us to develop a more robust and reusable solution that caters to a broader audience, ultimately enhancing the overall extension experience. The new issue for the "ScrollToTop" component has been created, and you can follow its progress and contribute to discussions here: #25.

We appreciate your understanding and continued engagement. If you have any questions or suggestions, please feel free to share them in the new issue.
